pandas/core/arrays/integer.py

```diff
@@ -205,7 +205,14 @@ def coerce_to_array(values, dtype, mask=None, copy=False):
             mask = mask.copy()
         return values, mask
 
-    values = np.array(values, copy=copy)
+    if isinstance(values, list):
+        values = np.array(values, dtype=object)
+
+    elif isinstance(values, np.ndarray):
```

Comment:
This may be missing a copy, when `copy=True`?

Reply:
It's handled now.